.storybook/webpack.config.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+/* eslint-disable no-underscore-dangle */
+
+/* eslint-disable no-param-reassign */
+import dotenv from 'dotenv';
+import path from 'path';
+import {DefinePlugin} from 'webpack';
+import type {Configuration, RuleSetRule} from 'webpack';
+
+type CustomWebpackConfig = {
```

Comment:
Could you check if there is already some `webpack` type we can use for it?

Reply:
We are already using Configuration from webpack. CustomWebpackConfig was created to workaround some type issues, without this type we would need to have a lot of condition to TS comply with the correct typing.
I find Configuration from webpack a bit too generic for this specific case.